Explorer/Assets/Scripts/Global/Dynamic/MainSceneLoader.cs

```diff
@@ -175,6 +179,14 @@ private async UniTask InitializeFlowAsync(CancellationToken ct)
 
                 bool shouldEnableLandscape = enableLandscape;
 
+                var hybridSceneParams = new HybridSceneParams();
+                if (useRemoteAssetBundles && initialRealm == InitialRealm.Localhost)
+                {
+                    hybridSceneParams.EnableHybridScene = true;
```

Comment:
I think we can remove this flag and assume it's always true. Is there a scenario when you don't want to enable it?

Reply:
Agreed. Removing it from the if and setting it always true